signer/src/stacks/contracts.rs

```diff
@@ -518,20 +531,226 @@ impl AsContractCall for AcceptWithdrawalV1 {
     /// Validates that the accept-withdrawal-request satisfies the
     /// following criteria:
     ///
-    /// 1. That the transaction with the associated request_id is stored as
-    ///    an event on the canonical Stacks blockchain.
-    /// 2. That the transaction associated with the outpoint has been
-    ///    confirmed on the canonical bitcoin blockchain.
-    /// 3. That the signer bitmap matches the signer decisions stored in
-    ///    this signer's database.
-    /// 4. That the `tx_fee` matches the amount spent to the bitcoin miner.
-    async fn validate<S>(&self, _db: &S, _ctx: &ReqContext) -> Result<(), Error>
+    /// 1. That the smart contract deployer matches the deployer in our
```

Comment:
[Suggestion] Pretty much the same doc comments I had on the other PR :)

Reply:
Done in https://github.com/stacks-network/sbtc/pull/555/commits/002fc518b6f1e828b0c2ab885f827cdc34baaa7d.